docs/user_guide/specifying_finance_parameters.md

```diff
@@ -1,282 +1,155 @@
-        cash_onhand_months: 1
-        admin_expense: 0.00 #administrative expense as a fraction of sales
-    #default parameters for capital items unless specified in tech_config
-    capital_items:
-      depr_type: "MACRS" #depreciation method for capital items, can be "MACRS" or "Straight line"
-      depr_period: 5 #depreciation period for capital items in years.
-      refurb: [0.] #refurbishment schedule, values represent the replacement cost as a fraction of the CapEx
-
-  # To adjust costs for technologies to target_dollar_year
-  cost_adjustment_parameters:
-    target_dollar_year: 2022
-    cost_year_adjustment_inflation: 0.025
+  finance_groups:
+    commodity: "hydrogen"
+    finance_model: "ProFastComp"
+    model_inputs:
+      discount_rate: 0.08
 ```
 
-This approach also relies on data from `plant_config`:
-- `plant_life`: used as the `operating life` ProFAST parameter
-
-
-```{note}
-`inflation_rate` is used to populate the escalation and inflation rates in ProFAST entries with a value of 0 corresponding to a *nominal analysis*.
+Outputs are named:
+```
+finance_subgroup_default.<finance_model_output>
 ```
 
-(finance:pf_params_opt)=
-## Providing Finance Parameters: ProFAST format
-
-```{note}
-To avoid errors, please check that `plant_config['plant']['plant_life']` is equal to `plant_config['finance_parameters']['model_inputs']['params']['operating life']`. Or remove `operating life` from the finance parameter inputs.`
+Examples:
+- [Example 7](https://github.com/NREL/H2Integrate/blob/develop/examples/07_run_of_river_plant/plant_config.yaml)
 
 
-| plant config parameter | equivalent `params` parameter |
-| -------- | ------- |
-| `plant['plant']['plant_life']` | `operating life` |
-```
+(finparams:singlemodelsubgroups)=
+## Single Finance Model With Subgroups
+If `finance_groups` contains a single model definition, you may split technologies into multiple subgroups. Each subgroup defines its own `commodity` and list of `technologies` but uses the shared finance model.
 
-Below is an example of the `finance_parameters` section of `plant_config` if using ProFAST input format to specify financial parameters:
+In this case you see that the commodity is not defined within the `finance_groups` and is instead defined within the `finance_subgroups`.
 
+General format:
 ```yaml
 finance_parameters:
-  finance_model: "ProFastComp"
-  model_inputs:
-    params: !include  "profast_params.yaml" #Finance information
-    capital_items: #default parameters for capital items unless specified in tech_config
-      depr_type: "MACRS" ##depreciation method for capital items, can be "MACRS" or "Straight line"
-      depr_period: 5 #depreciation period for capital items
-      refurb: [0.]
-  cost_adjustment_parameters:
-    target_dollar_year: 2022
-    cost_year_adjustment_inflation: 0.025 # used to adjust costs for technologies to target_dollar_year
+  finance_groups:
```

Comment:
In this scenario, is only hydrogen part of the financial calculations? (i.e. if you don't specify electricity, does LCOE get calculated, or included in the hydrogen calculation?)